### DISCUSSION TITLE
Continous Build

### DISCUSSION THREAD

**Discussion by asbachb** [support-and-care/maven-support-and-care#15] [Ideas for Funding]
I was always annoyed that every IDE use their own "compile on save"-Feature. This leads to different behaviors on different IDEs.

I think it would be quite cool to have a continuous build mode, where maven is watching for changes (add/remove/update) and delegates these changes to the plugins which could handle these events accordingly.

This would enable to move many things into the maven build like formatting or organize of imports or have a generic way to support continuous development like already available within the quarkus plugin.

**Comment by slawekjaranowski** [comment]
We use spotless plugin for check and formatting code.

There are also formatter for IDE which can be imported by developers ... I use it with InteliJ
https://github.com/apache/maven-shared-resources/tree/HEAD/src/main/resources/config

https://maven.apache.org/developers/conventions/code.html
